chemprop/nn/utils.py

```diff
@@ -59,4 +59,19 @@ def __call__(self, outputs):
 
         transformed_outputs = torch.from_numpy(self.output_scaler.inverse_transform(outputs))
 
-        return transformed_outputs
\ No newline at end of file
+        return transformed_outputs
+
+
+class InputTransform:
+
+    def __init__(self, input_scalers: dict[str, StandardScaler] | None = None):
+        self.input_scalers = input_scalers
+
+    def __call__(self, dataset):
+        KEYS = {"X_d", "V_f", "E_f", "V_d"}
+
+        for key in KEYS:
+            scaler = self.input_scalers.get(key, None) if self.input_scalers else None
+            if scaler is not None:
+                dataset.normalize_inputs(key, scaler)
```

Comment:
does this even need to be a class actually? Can't we just define as the underlying dictionary of key -> scaler? That is, assuming we've saved a `dict[str, StandardScaler]` after training, then during prediction, we should be able to just do:

```python
feature_to_scaler: dict[str, StandardScaler] = ...
for feature in feature_to_scaler:
    dataset.normalize_inputs(feature, scaler)
```